CONTRIBUTING.md

```diff
@@ -21,22 +21,46 @@ The core team will be monitoring for pull requests. When we get one, we'll run s
 *Before* submitting a pull request, please make sure the following is done…
 
 1. Fork the repo and create your branch from `master`.
-
+   A guide on how to fork a repository: https://help.github.com/articles/fork-a-repo/
+   
+   Open Git Bash (or Git Shell), and type: 
    ```sh
    git clone https://github.com/<your_username>/jest
    cd jest
    git checkout -b my_branch
    ```
+   Note:
+   Replace ``` <your_username> ``` with your github username
+   Replace ``` <my_branch> ``` with ``` <your_username>/jest ``
 
 2. Jest uses [Yarn](https://code.facebook.com/posts/1840075619545360)
    for running development scripts. If you haven't already done so,
    please [install yarn](https://yarnpkg.com/en/docs/install).
 
 3. Run `yarn install`.
-
+   On Window's: 
+   To install [Yarn] and to test what the version, switch to a command line.
+   To install [Yarn] on windows you may need to download either node.js or Chocolatey
```

Comment:
This will be displayed in one line. Please check how things look by clicking "View" button
 
<img width="999" alt="screen shot 2017-04-05 at 18 57 37" src="https://cloud.githubusercontent.com/assets/5106466/24717145/c3908f4c-1a31-11e7-96ac-453fc8d63a49.png">

Also, shouldn't [Yarn] be a link? I'd rather point to [this page](https://yarnpkg.com/en/docs/install#windows-tab
) for installation instructions